### DISCUSSION TITLE
r.FillBytes undefined

### DISCUSSION THREAD

**Discussion by bxwqsbkl** [golang-jwt/jwt#191] [Q&A] [answered]
# github.com/golang-jwt/jwt
/go/pkg/mod/github.com/golang-jwt/jwt@v3.2.2+incompatible/ecdsa.go:135:4: r.FillBytes undefined (type *big.Int has no field or method FillBytes)
/go/pkg/mod/github.com/golang-jwt/jwt@v3.2.2+incompatible/ecdsa.go:136:4: s.FillBytes undefined (type *big.Int has no field or method FillBytes)

**Comment by oxisto** [comment, marked as answer]
You need at least Go 1.15 to use this library, recommended and supported is at least Go 1.16, as specified in the go.mod.
